package.json

```diff
@@ -21,11 +21,11 @@
     "gulp-clean-css": "4.2.0",
     "gulp-concat": "^2.6.1",
     "gulp-minify": "^3.1.0",
-    "gulp-sass": "^4.1.0",
+    "gulp-sass": "^5.0.0",
```

Comment:
Preferably don't use versions that are the first e.g. `5.0.0` or `1.0.0`, etc. If there isn't a `5.1` or at least a `5.0.1` version available, I would prefer to stick with any `4.x` version.